### ISSUE TITLE
Multi dimensional image in Napari

### ISSUE THREAD

**Issue by manerotoni** [NEUBIAS/training-resources#531]
Hello @tibuch and @tischi, and everyone else,

we should definitely have a module on how to look at 5D images in Napari. It is not as plug and play as with IJ.
What do you think? 

Antonio

**Comment by manerotoni** [comment]
This should probably be in  the first module. Digital image basics. 
This is related to the issue https://github.com/NEUBIAS/training-resources/issues/453

**Comment by manerotoni** [comment]
see also https://ilovesymposia.com/2019/10/24/introducing-napari-a-fast-n-dimensional-image-viewer-in-python/
```
viewer = napari.Viewer()
viewer.add_image(mitosis[:, :, 0, :, :], colormap='magenta',
                 name='aurora-B', scale=[1, 10, 1, 1],
                 contrast_limits=[1500, 6500])
viewer.add_image(mitosis[:, :, 1, :, :], colormap='green',
                 name='tubulin', scale=[1, 10, 1, 1],
                 contrast_limits=[1600, 16000])
```

**Comment by manerotoni** [comment]
Addressed in PR #533
